src/routes/template.js

```diff
@@ -3,16 +3,16 @@ import auth from "../middleware/auth";
 
 export default (app) => {
     /* Route for Template Create  */
-	app.route("/template/add").post(auth.requiresAdminOrHr, template.create);
+    app.route("/template/add").post(auth.requiresAdminOrHr, template.create);
 
     /* Route for Template update  */
-	app.route("/template/update/:templateId").put(auth.requiresAdminOrHr, template.update);
+    app.route("/template/update/:templateId").put(auth.requiresAdminOrHr, template.update);
```

Comment:
need to change in all routes, discuss with me. the routes are not done properly.